File: Iniciante/URI 1002.go
package main
 
import (
    "fmt"
)
 
func main() {
    var raio, area float64

    fmt.Scanf("%f", &raio)
    area = 3.14159 * (raio * raio)
    fmt.Printf("A=%.4f\n", area)
}

File: Iniciante/URI 1002.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var n = 3.14159;
var raio = parseFloat(lines.shift());
var area = n * (raio * raio);
   
console.log('A=' + area.toFixed(4));